#! /bin/sh /usr/share/dpatch/dpatch-run
## 05_glib_single_include.dpatch by Colin Watson <cjwatson@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Include <glib.h> rather than individual headers.
## DP: http://bugs.debian.org/665550

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' gtk-sharp2-2.12.10~/glib/glue/list.c gtk-sharp2-2.12.10/glib/glue/list.c
--- gtk-sharp2-2.12.10~/glib/glue/list.c	2009-01-07 16:54:06.000000000 +0000
+++ gtk-sharp2-2.12.10/glib/glue/list.c	2012-04-02 23:50:36.000000000 +0100
@@ -20,7 +20,7 @@
  */
 
 
-#include <glib/glist.h>
+#include <glib.h>
 
 /* Forward declarations */
 gpointer gtksharp_list_get_data (GList *l);
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' gtk-sharp2-2.12.10~/glib/glue/slist.c gtk-sharp2-2.12.10/glib/glue/slist.c
--- gtk-sharp2-2.12.10~/glib/glue/slist.c	2009-01-07 16:54:06.000000000 +0000
+++ gtk-sharp2-2.12.10/glib/glue/slist.c	2012-04-02 23:50:40.000000000 +0100
@@ -20,7 +20,7 @@
  */
 
 
-#include <glib/gslist.h>
+#include <glib.h>
 
 /* Forward declarations */
 gpointer gtksharp_slist_get_data (GSList *l);
